viewers/wicket/ui/src/main/java/org/apache/isis/viewer/wicket/ui/components/scalars/datepicker/TextFieldWithDateTimePicker.java

```diff
@@ -131,16 +133,21 @@ protected void onComponentTag(ComponentTag tag) {
 
         checkComponentTag(tag, "input");
         Attributes.set(tag, "type", "text");
+        
+        Attributes.addClass(tag, "datetimepicker-input");
+        Attributes.set(tag, "data-toggle", "datetimepicker");
+        Attributes.set(tag, "data-target", getMarkupId());
     }
 
     @Override
     public void renderHead(final IHeaderResponse response) {
         super.renderHead(response);
 
-        response.render(CssHeaderItem.forReference(new CssResourceReference(TextFieldWithDateTimePicker.class, "css/bootstrap-datetimepicker.css")));
+        response.render(CssHeaderItem.forReference(new CssResourceReference(TextFieldWithDateTimePicker.class, "css/fa-patch.css")));
```

Comment:
See how we can get rid of this, maybe submit a patch to the datetime-picker project?